guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,673 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9a65c44a",
+   "metadata": {},
+   "source": [
+    "# Introduction to ArcGIS Hub - Basic features\n",
+    "\n",
+    "ArcGIS Hub is a cloud-based engagement platform that enables organizations to communicate more effectively with their communities. ArcGIS Hub Basic is available with all ArcGIS Online subscriptions. Using Hub Basic, you can create unlimited websites and pages using the site editor and build a catalog of searchable content for each of your sites using the site's content library. Click [here](https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources) to learn more about ArcGIS Hub."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cc8498c",
+   "metadata": {},
+   "source": [
+    "Note: The pattern to add, search, get, update, delete `Sites` and `Pages` is identical to that of __Initiatives__ (insert link to premium nb) as demonstrated here. \n",
+    "\n",
+    "In this notebook, we will take a look at examples of how to\n",
+    "* Create a site and pages\n",
+    "* Clone a site within the same (Hub Basic) org, ArcGIS Enterprise orgs, and ArcGIS Hub Premium orgs\n",
+    "* Link and unlink pages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "69b5b63f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from arcgis.gis import GIS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "972dce0a",
+   "metadata": {},
+   "source": [
+    "First, we will sign-in to an organization with Hub Basic capabilities for the purpose of demonstrating these workflows.\n",
+    "\n",
+    "Note: In order to replicate these workflows, you may have to sign-in using your own credentials for your ArcGIS organization. This notebook is only designed for demonstration purposes. You may also want to verify the features available to you, per your ArcGIS Hub licensing.\n",
+    "\n",
+    "Below are examples for working with Initiatives and Events using __ArcGIS Premium__ (insert live URL), site and page layouts, and theme editing using __Enterprise Sites__ (insert live URL). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "de1bf233",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter password: ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "gis1 = GIS(\"https://www.arcgis.com\", \"MMajumdar_geosaurus\")"
+   ]
```

Comment:
* replace this login with the appropriate `GIS(profile="home")` or `GIS(profile="your_online_profile")`